gradle-plugin-integration-test/multiplatform/build.gradle.kts

```diff
@@ -73,6 +72,7 @@ kotlin {
         all {
             compilations.all {
                 kotlinOptions {
+                    allWarningsAsErrors = false
```

Comment:
The suppression of `OPTIONAL_DECLARATION_USAGE_IN_NON_COMMON_SOURCE` _itself_ throws a compilation warning that can't (as far as I can tell) be excluded with a compiler arg. Gets a shrug from me.